server/knexfile.js

```diff
@@ -0,0 +1,37 @@
+require('dotenv').config({ path: '../.env' });
+module.exports = {
+  development: {
+    client: 'pg',
+    connection: {
+      host: process.env.DB_HOST || 'localhost',
+      database: process.env.DB_NAME,
+      user: process.env.DB_USER || 'postgres',
+      password: process.env.DB_PASS,
+    },
+    migrations: {
+      directory: './data/migrations',
+    },
+    seeds: { directory: './data/seeds' },
+  },
+  test: {
+    client: 'sqlite3',
+    connection: ':memory:',
```

Comment:
the test database is sqlite instead of postgres because sqlite can be run in memory, making for easy setup and teardown for tests